.github/workflows/.continuous-integration.yml

```diff
@@ -0,0 +1,32 @@
+name: Test
+
+on: pull_request
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: actions/setup-ruby@v1
+      with:
+        ruby-version: '2.6.3'
+    - name: Set up Docker
+      run: docker network create test
+    - name: Set up Postgres
+      run: docker run -d --name pg --network test -e POSTGRES_USER=bouncer -p 5432:5432 postgres:11
+    - name: Build a new Docker image
+      run: docker build . --build-arg RAILS_ENV=test -t bouncer:test
+    - name: Set up a new container for testing
+      run: |
+        docker run -d --name test-container \
+          --network test \
+          -e RAILS_ENV=test \
+          -e DATABASE_CLEANER_ALLOW_REMOTE_DATABASE_URL=true \
+          -e DATABASE_URL=postgres://bouncer@pg:5432/bouncer \
+          bouncer:test /bin/bash -c "tail -f /dev/null"
+    - name: Set up the test database
+      run: |
+        docker run --network test -v $(pwd):/bouncer -e POSTGRES_USER=bouncer postgres:11 \
+        /bin/bash -c "cat bouncer/db/structure.sql | psql 'postgresql://bouncer@pg:5432/bouncer'"
```

Comment:
This is a Rack app rather than a Rails one. The usual `rake db:setup` command is not available. I spent some time trying to plumb it in but couldn't do it without something depending on a Rails library. Taking the steer from the Rakefile which uses the psql CLI to reset the database, I thought it better to take this approach.

The approach here is to avoid installing psql dependencies into the base image. We can do this by starting a temporary postgres container, loading in the application files (for structure.sql), connecting to the running database and running a migration.

For some reason it wants to name the database `bouncer` rather than `transition_test` which is configured in database.yml. This is odd in itself since transition is a different application. Given it is only for CI, and we can inject our database name through DATABASE_URL, I'm not too worried what it's called in this context.